agent/lib/kontena/workers/weave_worker.rb

```diff
@@ -55,35 +55,9 @@ def on_container_event(topic, event)
 
     # @param [Docker::Container] container
     def weave_attach(container)
-      config = container.info['Config'] || container.json['Config']
```

Comment:
:+1: for fixing this `.info || .json` thing, because this is called with both `Docker::Container.get` and `Docker::Container.all.each`